src/model.jl

```diff
@@ -45,12 +45,16 @@ end
 
 struct Production
     sector::Symbol    
-    elasticity::Float64 # TODO Make Union{Float64,Expr}
+    elasticity::Union{Float64,Expr}
     output::Symbol
-    output_quantity::Float64 # TODO Make Union{Float64,Expr}
+    output_quantity::Union{Float64,Expr}
     inputs::Vector{Input}
 end
 
+function Production(sector::Symbol, elasticity::Number, output::Symbol, output_quantity::Number, inputs::Vector{Number})
```

Comment:
We actually have to find a better way for this, because now if you use an `Int` for one of them, you can't use an expression for the other anymore... Maybe something like this:
```julia
function Production(sector::Symbol, elasticity::Union{Number,Expr}, output::Symbol, output_quantity::Union{Number,Expr}, inputs::Vector{Input})

  if elasticity isa Number
    elasticity = convert(Float64, elasticity)
  end

  if output_quantity isa Number
    output_quantity = convert(Float64, output_quantity)
  end

  return Production(sector, elasticity, output, output_quantity, inputs)
end
```

Also, I don't think you wanted to change the element type for `inputs`, right?

Reply:
Ok, yes, thanks. Figured this out and looks good.